### DISCUSSION TITLE
Is there a easy way to make uBlockOrigin "invisible" to websites?

### DISCUSSION THREAD

**Discussion by Buffmutti** [uBlockOrigin/uBlock-issues#2059] [Q&A] [answered]
In the last few weeks i realize that more and more websites seems to "see" that i'm using a adblocker. On Web.de the detection is a bit wonky, 80% of the time the website will tell you that you don't use a adblocker, but in 20% of the time they will ask you to deactivate your adblocker (simply reload the website and suddenly the website can't see the adblocker anymore). On IGG.com the detection seems to be flawless, the adblocker will always be detected (and since EVERY click on any spot in that website leads to you being redirected to some other websites with very aggressiv ads and scams you will do everything but disable your adblocker).

Is there a way to make uBlockOrigin "invisible" on websites?

**Comment by gorhill** [comment, marked as answer]
Just report such instances using the report button in the popup panel, filters will be created for these, as is the case for countless other sites.
